static/app/gettingStartedDocs/flutter/flutter.tsx

```diff
@@ -452,10 +459,8 @@ const replayOnboarding: OnboardingConfig<PlatformOptions> = {
     },
   ],
   verify: getReplayVerifyStep({
-    replayOnErrorSampleRateName:
-      'options\u200b.experimental\u200b.sessionReplay\u200b.onErrorSampleRate',
-    replaySessionSampleRateName:
-      'options\u200b.experimental\u200b.sessionReplay\u200b.sessionSampleRate',
+    replayOnErrorSampleRateName: 'options\u200b.replay\u200b.onErrorSampleRate',
+    replaySessionSampleRateName: 'options\u200b.replay\u200b.sessionSampleRate',
```

Comment:
```suggestion
    replayOnErrorSampleRateName: 'options.replay.onErrorSampleRate',
    replaySessionSampleRateName: 'options.replay.sessionSampleRate',
```

Reply:
not sure where this comes from but I guess it's not needed/intended?

Reply:
am also not sure but it is like that for other onboarding docs so I wouldn't dare to change it https://github.com/getsentry/sentry/blob/bdfece436c1fd0fa0b01c7b59977eb5d63986ae2/static/app/gettingStartedDocs/android/android.tsx#L467-L468